client/trino-jdbc/src/main/java/io/trino/jdbc/NonRegisteringTrinoDriver.java

```diff
@@ -90,12 +89,14 @@ public Connection connect(String url, Properties info)
     private Call.Factory instrumentClient(OkHttpClient client)
     {
         try {
-            OpenTelemetry openTelemetry = GlobalOpenTelemetry.get();
-            return OkHttpTelemetry.builder(openTelemetry).build().newCallFactory(client);
+            return OkHttpTelemetry
+                    .builder(GlobalOpenTelemetry.get())
```

Comment:
An alternative would be to allow providing the `OpenTelemetry` object explicitly through a connection property. We already have connection properties that can accept objects. WDYT?

Reply:
what's the reason?

Reply:
We had users reporting on Slack this was causing issues for them, since `GlobalOpenTelemetry.get()` can set the global object to a noop implementation, which cannot be changed later. This forces users to initialize OTEL before using the Trino driver. It might not be a big deal, but IMO the benefits from instrumentation getting enabled automatically are less important than avoiding unexpected behavior like this.

Reply:
Yeah, I've figured

Reply:
PTAL now

Reply:
I suggested adding a property, to be able to configure it in applications using the driver manager, Hikari or others were creating an instance of the driver is not possible.

Reply:
Property isn't a good fit since it couples a lot of classes with OpenTelemetry

Reply:
What about a custom uri like for postgresql ? 

jdbc:otel:postgresql://${RDS_ENDPOINT}/${RDS_DATABASE}

Reply:
Looks like it's possible to call methods of drivers already registered in the driver manager, like here: https://github.com/open-telemetry/opentelemetry-java-instrumentation/blob/main/instrumentation/jdbc/library/src/main/java/io/opentelemetry/instrumentation/jdbc/OpenTelemetryDriver.java#L206

So this is fine, thanks!

Reply:
I can try to follow-up with examples in docs.